lib/ansible/plugins/lookup/hashi_vault.py

```diff
@@ -56,27 +56,27 @@
 
 EXAMPLES = """
 - debug:
-    msg: "{{ lookup('hashi_vault', 'secret=secret/hello:value token=c975b780-d1be-8016-866b-01d0f9b688a5 url=http://myvault:8200')}}"
+    msg: "{{ lookup('hashi_vault', secret=secret/hello:value, token=c975b780-d1be-8016-866b-01d0f9b688a5, url=http://myvault:8200) }}"
```

Comment:
I don't think this is valid Jinja2 syntax. Do you mean:
```suggestion
    msg: "{{ lookup('hashi_vault', secret='secret/hello:value', token='c975b780-d1be-8016-866b-01d0f9b688a5', url='http://myvault:8200') }}"
```
?